package.json

```diff
@@ -1,17 +1,20 @@
 {
   "name": "babel-plugin-react-cssmoduleify",
-  "version": "0.9.5",
+  "version": "1.0.0-beta",
   "description": "Babel plugin to transform traditional classNames to CSS Modules",
   "main": "lib/index.js",
-  "jsnext:main": "index.js",
+  "jsnext:main": "src/index.js",
   "scripts": {
-    "build": "babel index.js -o lib/index.js",
+    "build": "babel src/ --out-dir lib/",
     "prepublish": "npm run build",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint .",
+    "compile-tests": "babel --presets es2015,react test/fixtures/jsx/*/ --out-dir test/fixtures/compiled",
```

Comment:
:)

I generated the `tests/fixtures/compiled` tests from the `test/fixtures/jsx` tests. In practice we’re actually using this plugin on the already compiled source (in our internal Reat components) so that is really the most important case for usage.